External CI: updated MIOpen dependencies

amd-jmacaran opened the pull request:
Successful build log: https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1795&view=results

danielsu-amd commented:
There are some warnings about cmake not finding roctracer and zstd in the flag step but it looks like it builds ok. We can add in roctracer, but I'm not sure about the zstd issue

https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1795&view=logs&j=88ba0832-789b-5bb9-0dd5-a1a7441d06cc&t=99afa8fd-74f8-55a4-27c4-738732a96958&l=74
https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1795&view=logs&j=88ba0832-789b-5bb9-0dd5-a1a7441d06cc&t=99afa8fd-74f8-55a4-27c4-738732a96958&l=97

amd-jmacaran commented:
> There are some warnings about cmake not finding roctracer and zstd in the flag step but it looks like it builds ok. We can add in roctracer, but I'm not sure about the zstd issue
> 
> https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1795&view=logs&j=88ba0832-789b-5bb9-0dd5-a1a7441d06cc&t=99afa8fd-74f8-55a4-27c4-738732a96958&l=74 https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1795&view=logs&j=88ba0832-789b-5bb9-0dd5-a1a7441d06cc&t=99afa8fd-74f8-55a4-27c4-738732a96958&l=97

Added roctracer and removed libzstd-dev, since that and adding zstd via apt did nothing. Build log below. The zstd references would likely be solved from modifying llvm-project build.

https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=1812&view=results
